frontend/src/bundles/common/icon-provider.ts

```diff
@@ -2,17 +2,47 @@ import { faEye, faEyeSlash } from '@fortawesome/free-regular-svg-icons';
 import {
     faArrowDown,
     faArrowUp,
+    faBasketShopping,
+    faBriefcase,
+    faCar,
     faChevronDown,
+    faCloudArrowUp,
+    faCopy,
+    faGasPump,
+    faGear,
+    faGift,
+    faGraduationCap,
+    faLandmark,
+    faPaw,
+    faPen,
     faPlus,
+    faSackDollar,
+    faStop,
+    faTrash,
 } from '@fortawesome/free-solid-svg-icons';
 
 const iconProvider = {
```

Comment:
I would prefer enum instead of object. And passed it as a type to findIcon function.